java/examples/pom.xml

```diff
@@ -53,16 +53,33 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
       <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Isn't it better to put the exclusions in the main `pom.xml` file so that we don't have to define them for every module?

Reply:
Great, let me try. Thank you @zabetak